src/script/user/UserRepository.ts

```diff
@@ -482,10 +489,10 @@ export class UserRepository {
         });
     };
 
-    const chunksOfUserIds = chunk(userIds, z.config.MAXIMUM_USERS_PER_REQUEST);
+    const chunksOfUserIds = chunk(userIds, (z as any).config.MAXIMUM_USERS_PER_REQUEST) as string[][];
```

Comment:
If you are courageous enough, you can import `config.js` directly (but can be done later, no problem)